labelbox/schema/send_to_annotate_params.py

```diff
@@ -0,0 +1,59 @@
+import sys
+
+from typing import Optional, Dict
+
+from labelbox.schema.conflict_resolution_strategy import ConflictResolutionStrategy
+
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+
+class SendToAnnotateFromCatalogParams(TypedDict):
+    """
+    Extra parameters for sending data rows to a project through catalog. At least one of source_model_run_id or
+    source_project_id must be provided.
+
+    :param source_model_run_id: Optional[str] - The model run to use for predictions. Defaults to None.
+    :param model_run_ontology_mapping: Optional[Dict[str, str]] - A mapping of ontology ids to ontology ids. Defaults to
```

Comment:
Should we rename these to
`predictions_ontology_mapping` and `annotations_ontology_mapping`, to stay consistent with `predictions` vs `annotations` terminology?

Reply:
Also I think `A mapping of feature schema ids to feature schema ids` is more accurate